docker-compose.ubuntu.example.yml

```diff
@@ -17,7 +17,9 @@ services:
     volumes:
        - ./example/haproxy/haproxy.cfg:/etc/haproxy/haproxy.cfg
        - ./example/haproxy/pem/pubkey.pem:/etc/haproxy/pem/pubkey.pem
+       - ./example/haproxy/pem/pubkey2.pem:/etc/haproxy/pem/pubkey2.pem
        - ./example/haproxy/pem/test.com.pem:/etc/haproxy/pem/test.com.pem
+       - ./lib/jwtverify.lua:/usr/local/share/lua/5.4/jwtverify.lua
```

Comment:
In the existing repo, `jwtverify.lua` gets installed by `example/haproxy/install.sh` line 120:

```bash
install_luaoauth() {
    printf "\r[+] Installing haproxy-lua-oauth\n"
    if [ ! -e $lua_dep_dir ]; then
        mkdir -p $lua_dep_dir;
    fi;

    cp $CWD/lib/*.lua $lua_dep_dir
}
```

which test the install script. What's the reason to overwrite that installed lua file with a Docker volume?